src/components/ResultList.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import cityImg from '../city.png'
+
+export interface City {
+    id: number,
+    ascii: string,
+    name: string
+}
+
+
+export const ResultList: React.FunctionComponent<{ cities: City[] }> = ({ cities }) => {
+    return <div className="ResultList">
+        <ol>
+            {cities.map((c) => <li key={c.id}>
+                <img src={cityImg} alt="City logo"/>
+                <span>Country: {c.name}</span>

```

Comment:
```suggestion
                <span>Country: {city.country}</span>
```

But the endpoint seems to provide a code (ex.: US), maybe we may do some mapping somewhere else to display (United States of America)?